### ISSUE TITLE
"/usr/bin/python: No module named mssqlcli"

### ISSUE THREAD

**Issue by yooakim** [dbcli/mssql-cli#101]
I've followed the installation instructions and when I try to run `mssql-cli` I get the subject error message.

Installation went OK so I am not sure what is wrong? I'm using Ubuntu 17.10

```
joakim@XPS:~$ pip install mssql-cli
Requirement already satisfied: mssql-cli in /usr/local/lib/python3.6/dist-packages
Requirement already satisfied: cli-helpers<1.0.0,>=0.2.3 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: click>=4.1 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: Pygments>=2.0 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: configobj>=5.0.6 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: future>=0.16.0 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: enum34>=1.1.6 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: wheel>=0.29.0 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: prompt-toolkit<1.1.0,>=1.0.10 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: applicationinsights>=0.11.1 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: humanize>=0.5.1 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: sqlparse<0.3.0,>=0.2.2 in /usr/local/lib/python3.6/dist-packages (from mssql-cli)
Requirement already satisfied: backports.csv>=1.0.0 in /usr/local/lib/python3.6/dist-packages (from cli-helpers<1.0.0,>=0.2.3->mssql-cli)
Requirement already satisfied: terminaltables>=3.0.0 in /usr/local/lib/python3.6/dist-packages (from cli-helpers<1.0.0,>=0.2.3->mssql-cli)
Requirement already satisfied: six in /usr/lib/python3/dist-packages (from configobj>=5.0.6->mssql-cli)
Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/dist-packages (from prompt-toolkit<1.1.0,>=1.0.10->mssql-cli)
```

**Comment by yooakim** [comment]
My bad, it says in the docs you need to ensurfe you have the correct Pythin env setup.

**Comment by trumpi** [comment]
I'm having the same problem, but on Mac OS. I'm not sure which docs you are referring to, but I cannot find anything that refers to a Python environment in the Mac OS docs. Can anyone help me?

**Comment by MrMeemus** [comment]
@trumpi, Can you please share the following info:
- Commands used to install mssql-cli
- Python3 and pip3 installed?

**Comment by trumpi** [comment]
Hi @MrMeemus 

I think I've solved the problem, but I admit that I probably did it by accident since I'm not an expert with Python. It appears as though I had both Python 2 and Python 3 installed. Even worse than that, there were two versions of Python 2 installed. I have no idea how that happened.

I resolved the problem by uninstalling all the Pythons using Homebrew. I then installed `python@2`. I later upgraded my vim which installed Python 3 as well.

Somehow, this fixes it and I'm able to use mssql-cli.

**Comment by m3asmi** [comment]
fixing it by installing by pip2
`pip2 install mssql-cli`

**Comment by EvanCarroll** [comment]
I'm totally confused. I'm also having this problem. Why are people using pip2, and python2? Does the cli not work with python3?

**Comment by pensivebrian** [comment]
mssql-cli supports both python 2.7 and 3.4+ and we leave it up to the user choose. For most Linux distributions, pip and python defaults to 2. I think Red Hat 8 (in beta) will be the first big distro to default to 3.

We generally see this error when multiple versions of python are installed, but pip runs from one location, and python from another. Running 'whereis python' or 'whereis pip' on the command line can be used to confirm the locations.

**Comment by eliot1785** [comment]
I'm also having this issue. On my Mac like most Macs I think, pip and python refer to 2.x, the system-provided python. I also have Python3 installed using homebrew, so pip3 and python3 refer to that. I installed mssql-cli using pip3 and got the aforementioned issue when I tried to run it. 

Note that this wasn't a case of "pip runs from one location, and python from another." Both pip and python refer to the system pip/python, I just tried to install using a different python. Maybe the issue is that you're calling out to python from your program and getting the system one?

Installing using pip (i.e. referring to python 2.7) worked, though I wasn't enthusiastic about having to sudo / modify the system-provided python.

Is this a different project from the MSSQL "command line utilities" that were released to Homebrew? https://github.com/Microsoft/homebrew-mssql-release
I installed them as well, but they don't seem to contain mssql-cli. Does that mean you're still working on a homebrew package for mssql-cli?
